.eslintrc

```diff
@@ -43,7 +43,8 @@
     "handle-callback-err": 0,
     "indent": [
       2,
-      "tab"
+      "tab",
+      { SwitchCase: 1 }
```

Comment:
I could drop this but otherwise it requires switch statements to be formatted like:
```
switch(a) {
case "blah":
default:
    return true;
}
```